src/TestSuite/IntegrationTestTrait.php

```diff
@@ -263,13 +270,27 @@ public function configApplication($class, $constructorArgs)
      * compatible token to be added to request data. This
      * lets you easily test actions protected by SecurityComponent.
      *
+     * @param string[] $unlockedFields List of fields that are excluded from field validation.
      * @return void
      */
-    public function enableSecurityToken()
+    public function enableSecurityToken($unlockedFields = [])
     {
+        $this->setUnlockedFields($unlockedFields);
+
         $this->_securityToken = true;
     }
 
+    /**
+     * Set list of fields that are excluded from field validation.
+     *
+     * @param string[] $unlockedFields List of fields that are excluded from field validation.
+     * @return void
+     */
+    public function setUnlockedFields($unlockedFields = [])
```

Comment:
Not a fan of public API changes in bugfix release even if it only affects testsuite. Let's at least add a `@since` tag to the docblock.

Reply:
The argument should be typehinted as `array`.